.travis.yml

```diff
@@ -31,6 +31,9 @@ before_install:
 install:
   - travis_retry mvn -B install clean -U -DskipTests=true
 
+after_success:
+  - mvn jacoco:report coveralls:report
+
```

Comment:
It runs just once, for the first job. Of course, coverage values could actually differ, but I think that is beyond coveralls.